editor/plugins/spatial_editor_plugin.h

```diff
@@ -137,10 +138,48 @@ class EditorSpatialGizmo : public SpatialGizmo {
 	~EditorSpatialGizmo();
 };
 
+class ViewportRotationControl : public Control {
+	GDCLASS(ViewportRotationControl, Control);
+
+	struct Axis2D {
+		Vector2 screen_point{ 0, 0 };
```

Comment:
This line causes some error during the CI build. The `Vector2` constructor is not marked as `noexcept` and that trips the compiler somehow. Adding the `noexcept` keyword to the `Vector2` constructor fixes the build, but I'm not sure it's the right fix or if I'm just using list-initializers wrongly.